test/SPI/private_swiftinterface.swift

```diff
@@ -29,16 +29,16 @@
 
 /// Test the textual interfaces generated from this test.
 // RUN: %target-swift-frontend -typecheck %s -emit-module-interface-path %t/Main.swiftinterface -emit-private-module-interface-path %t/Main.private.swiftinterface -enable-library-evolution -swift-version 5 -I %t -module-name Main
-// RUN: %FileCheck -check-prefix=CHECK-PUBLIC %s < %t/Main.swiftinterface
-// RUN: %FileCheck -check-prefix=CHECK-PRIVATE %s < %t/Main.private.swiftinterface
+// RUN: cat %t/Main.swiftinterface | %FileCheck -check-prefix=CHECK-PUBLIC %s
```

Comment:
Why use `cat` instead of `<` here, out of curiosity? FWIW, switching to `--input-file` instead would improve debugging somewhat (that way FileCheck will print the path to the file when there are mismatches)

Reply:
That was it, I prefer that style now to see the name of the file in the outputs. I'll take a look at `--input-file`, maybe it will be worth switching style again.